frontend/wallet/schema.graphql

```diff
@@ -43,50 +13,80 @@ type SyncUpdate {
 
 type SnarkWorker {
   key: String! @examples(values: ["PUB_KEY_E9873DF4453213303DA61F2", "PUB_KEY_F1173DF4453213303DA61F2"])
-  fee: String! @fake(type: money)
+  fee: String! @fake(type: number)
 }
 
-type SnarkFee {
-  snarkCreator: String! @examples(values: ["PUB_KEY_E9873DF4453213303DA61F2", "PUB_KEY_F1173DF4453213303DA61F2"])
-  fee: String! @fake(type: money)
+type UserCommand {
+  id: String!
+  isDelegation: Boolean!
+  nonce: Int!
+  date: String! @fake(type: pastDate)
+  from: String! @examples(values: ["PUB_KEY_E9873DF4453213303DA61F2", "PUB_KEY_F1173DF4453213303DA61F2"]) 
+  to: String! @examples(values: ["PUB_KEY_E9873DF4453213303DA61F2", "PUB_KEY_F1173DF4453213303DA61F2"])
+  amount: String! @fake(type: number)
+  fee: String! @fake(type: number)
+  memo: String @fake(type:hackerPhrase)
+}
+
+type FeeTransfer {
+  recipient: String! @examples(values: ["PUB_KEY_E9873DF4453213303DA61F2", "PUB_KEY_F1173DF4453213303DA61F2"])
+  fee: String! @fake(type: number)
+}
+
+type CoinbaseTransaction {
+	amount: String! @fake(type: number)
+	feeTransfer: FeeTransfer!
+}
+
+type BlockchainState {
+  date: String! @fake(type: pastDate)
+  snarkedLedgerHash: String!
+  stagedLedgerHash: String!
+  supplyIncrease: String! @fake(type: number)
```

Comment:
I was looking at the BlockchainState and SupplyIncrease is not a field of BlockchainState. Let's remove that field